styleguide/components/41-organisms/theme-footer/_theme-footer.scss

```diff
@@ -3,50 +3,156 @@
   @include theme('border-color', 'color-primary--lighten-4', 'theme-footer-border-bottom-color');
 
   width: calc(100% + 2 * #{$gutter-width});
-  margin-top: 4rem;
-  margin-right: -$gutter-width;
-  margin-left: -$gutter-width;
-  padding: 2.7rem 0 2.1rem;
+  margin-right: -1rem;
```

Comment:
Why isn't this done with $gutter-width or gutter()?

Reply:
Because the container padding isn't the gutter-width.
It's 1rem vs 1.2rem.